woocommerce/woocommerce-android
# [IPP] Update stripe SDK

Closes: #8881


### Description

Updates Stripe SDK version. This brings updates of the many other dependencies, including Room and ~~Kotlin std lib~~ (updated separately)

@oguzkocer @wzieba do you have any concerns regarding this?

### Testing instructions

* Run TTP and IPP (both supported readers) flows with and without `stripe_repo_url`
* Simulate crash and verify that it's send to Sentry
